lib/IRGen/IRGen.cpp

```diff
@@ -1162,6 +1163,12 @@ GeneratedModule IRGenRequest::evaluate(Evaluator &evaluator,
   // Run SIL level IRGen preparation passes.
   runIRGenPreparePasses(*SILMod, IGM);
 
+  if (!layoutStringsEnabled(IGM)) {
+    auto moduleName = IGM.getSwiftModule()->getRealName().str();
+    IGM.Context.Diags.diagnose(SourceLoc(), diag::layout_strings_blocked,
```

Comment:
Won't this diagnose for normal programs that don't enable CVWT, even if they are not on the block list?